src/borg/archive.py

```diff
@@ -723,15 +725,11 @@ class ChunksIndexError(Error):
         def chunk_decref(id, stats):
             nonlocal error
             try:
+                # note: this might do an async repo.delete(id)
                 self.cache.chunk_decref(id, stats)
```

Comment:
maybe adding a wait=True default to chunk_decref would be prettier, so we would need wait=False being given here and the async usage gets explicit without a comment.